src/iio/iio.c

```diff
@@ -172,10 +176,17 @@ mraa_iio_get_channel_data(mraa_iio_context dev)
     }
     closedir(dir);
 
-    // channel location has to be done in channel index order so do it afetr we
+    // channel location has to be done in channel index order so do it after we
```

Comment:
Please don't add unnecessary formatting changes, those just pollute the git history. This one is legitimate, but please split it out to a separate commit.